src/TLuaInterpreter.cpp

```diff
@@ -11247,35 +11249,6 @@ QString TLuaInterpreter::formatLuaCode(const QString &code)
     return result;
 }
 
-// No documentation available in wiki - internal function
-bool TLuaInterpreter::compileScript(const QString& code)
-{
-    lua_State* L = pGlobalLua;
-
-    int error = luaL_dostring(L, code.toUtf8().constData());
-    if (error != 0) {
-        std::string e = "no error message available from Lua";
-        if (lua_isstring(L, 1)) {
-            e = "Lua error:";
-            e += lua_tostring(L, 1);
-        }
-        if (mudlet::debugMode) {
-            TDebug(QColor(Qt::white), QColor(Qt::red)) << "LUA: code did not compile: ERROR:" << e.c_str() << "\n" >> 0;
-        }
-    } else {
-        if (mudlet::debugMode) {
-            TDebug(QColor(Qt::white), QColor(Qt::darkGreen)) << "LUA: code compiled without errors. OK\n" >> 0;
-        }
-    }
-    lua_pop(L, lua_gettop(L));
-
-    if (error == 0) {
-        return true;
-    } else {
-        return false;
-    }
-}
-
```

Comment:
You remove this whole function and don't even mention it in PR description. Why?

Reply:
I found it was unused and unneeded - but I forgot to document it in the commit message.

***Note to self: include something about this in the* Squash-and-merge *message.***

Reply:
Would not expect such change under a title "show profile indication in debug" at all - would you?

Reply:
Yeah, but it has `TDebug` messages that otherwise I'd have to revise - and why revise when one can nuke? :smiling_imp: